core/build.gradle

```diff
@@ -38,6 +38,22 @@ repositories {
     maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
 }
 
+sourceSets {
+    integrationTest {
+        kotlin {
+            compileClasspath += main.output + test.output
+            runtimeClasspath += main.output + test.output
+            srcDir file("${projectDir}/src/integration-test/kotlin")
```

Comment:
Jest to rozwiązanie dobre, ale widzę lepsze :p Moim zdaniem testy integracyjne mają tym większą wartość, im bardziej odseparowane są od aplikacji. Widziałem czasami jaki ludzie piszą testy integracyjne, które komunikują się za pomocą tego samego kodu co aplikacja - jak dla mnie, to takie bardziej pro jednostkowe są. Dążę do tego, że te testy integracyjne są częścią cora, mają świadomość czym jest core, że to jakaś składowa całości, a imo powinny mieć świadomość tylko, że istnieje aplikacja i udostępnia np. restowe api, czyli być podprojektem na poziomie cora. Nie ma jeszcze żadnych testów int. także może wybiegam z domysłami za daleko.

Może być jak jest, ma to wartość, ale może udałoby mi się Cię przekonać, że istnieją jeszcze lepsze podejścia, subiektywnie oczywiście, choć trudniejsze w implementacji, ale dające maksimum zaufania do własnego kodu. Na screen share bym pokazał o co mi chodzi dokładnie.